/yantra:
        gazebo_ros_control:
                pid_gains:
                        joint_0_1: &joint_0_1_gains
                                p: 100.0
                                i: 100.0
                                d: 500.0
                        joint_1_2: &joint_1_2_gains
                                p: 100.0
                                i: 100.0
                                d: 200.0
                        joint_2_3: &joint_2_3_gains
                                p: 100.0
                                i: 50.0
                                d: 100.0
                        joint_3_4: &joint_3_4_gains
                                p: 100.0
                                i: 50.0
                                d: 70.0
                        joint_4_51: &joint_4_51_gains
                                p: 1.0
                                i: 0.0
                                d: 0.0
        
        joint_state_controller:
                type: joint_state_controller/JointStateController
                publish_rate: 50
                joints: 
                        - joint_0_1
                        - joint_1_2
                        - joint_2_3
                        - joint_3_4
                        - joint_4_51
        
                          #one_link_controller:
                          #type: effort_controllers/JointPositionController
                          #joint: joint_0_1
                          #pid: {p: 1.0, i: 0.0, d: 0.0}

        yantra_arm_controller:
                type: effort_controllers/JointGroupPositionController
                joints:
                        - joint_0_1
                        - joint_1_2
                        - joint_2_3
                        - joint_3_4
                        - joint_4_51

                gains:
                        joint_0_1: {p: 100.0, i: 100.0, d: 500.0}
                        joint_1_2: {p: 100.0, i: 100.0, d: 500.0}
                        joint_2_3: {p: 100.0, i: 100.0, d: 500.0}
                        joint_3_4: {p: 100.0, i: 100.0, d: 500.0}
                        joint_4_51: {p: 100.0, i: 100.0, d: 500.0}
        
                 

